src/bit-systems/networking.ts

```diff
@@ -10,6 +10,17 @@ export const networkedQuery = defineQuery([Networked]);
 export const pendingMessages: Message[] = [];
 export const pendingJoins: StringID[] = [];
 export const pendingParts: StringID[] = [];
+export const softRemovedEntities = new Set<EntityID>();
 export function isNetworkInstantiated(eid: EntityID) {
   return createMessageDatas.has(eid);
 }
+
+let reticulum: StringID | undefined;
+export function isPinned(eid: EntityID) {
+  reticulum = reticulum || APP.getSid("reticulum");
```

Comment:
I removed the memoization because I don't actually know the impact and it seems like the kind of code someone would read and assume it was added with more thought / purpose than I actually put into it. https://github.com/mozilla/hubs/commit/7f51dc9a884821fef5b4c110ad3cacde1b15f014